doc_admin/AdminManual.adoc

```diff
@@ -1416,6 +1417,24 @@ By default, ALLOWPRINTOPENEDBILL is set to no.
 NOTE: An application restart is required to apply the modified setting.
 
 
+==== 3.1.36 PATIENTPHOTO
+
+The following table shows the default value and the allowed ones:
+
+[cols=",,",options="header",]
+|===
+|Key |Default Value |Valid Values
+|PATIENTPHOTO |DB |DB, filesystem path
+|===
+
+Open Hospital can save profile picture on DB or on filesystem. 
```

Comment:
Better wording: Open Hospital can save the user's profile picture in the database or on the file system.